_rmd/2019-03-10-RPyGeo-1_0_0.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "RPyGeo 1.0.0"
+date:  "`r format(Sys.time(), '%d %B, %Y')`"
+comments: true
+author: Marc Becker
+categories: r
+---
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+
+[DOWNLOADHERE]
+
+RPyGeo 1.0.0 has been released on CRAN. The RPyGeo package establishes an interface to the geoprocessing tools of ArcGIS from within R. ArGIS is a geographic information system developed by Esri. 
+
+The new RPyGeo package accesses the ArcPy Python side-package via the [reticulate](https://cran.r-project.org/web/packages/reticulate/index.html) package. ArcGIS provides almost all of it's geoprocessing tools via ArcPy. With RPyGeo these functions are available in R. Additonally RPyGeo provides utilitiy functions to create a seamless workflow between ArcGIS and R. 
+
+In 2015 Esri released the R-ArcGIS Bridge, a software that also connects ArcGIS and R. Essentially, the R-ArcGIS Bridge offers read, write and conversion functions to transfer data from ArcGIS to R and vice versa. The idea is to use the large number of R packages to solve spatial problems, which cannot be solved with ArcGIS alone. R scripts can be run as geospatial scripts with an interactive user inferface from within ArcGIS. However, no functionality is added for the R user. RPyGeo is able to run the geoprocessing tools of ArcGIS directly from the R session.      
+The update added many new features to the package. It is now possible to chain ArcPy functions togheter with the pipe operator. Help files for all ArcGIS functions can be viewed directly inside RStudio. Moreover, map algebra expressions can be used to modify raster objects.   
+
+For a detailed instruction on how to use RPyGeo we would like to refer to the [vignette](https://cran.r-project.org/web/packages/RPyGeo/RPyGeo.pdf). It includes a tutorial with all essential RPyGeo functions. 
+
+The devolepment of RPyGeo is now integrated into the r-spatial.org community. If you find any bugs please report them at our new development repository [r-spatial/RPyGeo](https://github.com/r-spatial/RPyGeo) or help us fix them via a pull request.
```

Comment:
> The devolepment of RPyGeo is now integrated into the r-spatial.org community. If you find any bugs please report them at our new development repository [r-spatial/RPyGeo](https://github.com/r-spatial/RPyGeo) or help us fix them via a pull request.

The source code of RpyGeo is now hosted on Github within the r-spatial organization.